libs/ui-watt/src/lib/foundations/color/storybook/color.stories.ts

```diff
@@ -0,0 +1,41 @@
+import { moduleMetadata, Story, Meta } from '@storybook/angular';
+
+import { ColorComponent } from './color.component';
+import { ColorModule } from './color.module';
+
+export default {
+  title: 'Foundations/Color',
+  component: ColorComponent,
+  decorators: [
+    moduleMetadata({
+      imports: [ColorModule],
+    })
+  ],
+} as Meta<ColorComponent>;
+
+//👇 We create a “template” of how args map to rendering
+const Template: Story<ColorComponent> = (args) => ({
+  props: args,
+});
+
+//👇 Each story then reuses that template
+export const Color = Template.bind({});
+Color.parameters = {
+  docs: {
+    source: {
+      code: ` // Usage from SCSS:
+@use '@energinet/watt' as watt;
+.my-element {
+  background: watt.getColor('COLOR NAME');
+  color: watt.getColorContrast('COLOR NAME-contrast');
+}
+
+// Usage from TypeScript:
+1. import { Colors, ColorHelperService } from '@energinet/watt';      
+1. Inject the ColorHelperService
+2. Use ColorHelperService.getColor(Colors.COLOR NAME); or ColorHelperService.getColorContrast(Colors.COLOR NAME);
```

Comment:
Question: Do we need a list of the available color names?

Reply:
I've done a copy to clipboard functionality now, when you hover the color samples. I will try take a look at how we can document the full-list, maybe we can make use of combodoc for that purpose

Reply:
I will give storybook-addon-cssprops a try

Reply:
Just saw that the variable names are on top of the colors. It looks fine as a first version. The only thing is aligning it with Stephanie. Maybe she'll need to update Figma accordingly.

Reply:
storybook-addon-cssprops printed out errors in the console due to wrong implementation in React, so that's not an option. If we wanna document it further down the road, we're properly forced to do something ourselves.